pagerdutyplugin/data_source_pagerduty_escalation_policy.go

```diff
@@ -26,6 +27,13 @@ func (*dataSourceEscalationPolicy) Schema(ctx context.Context, req datasource.Sc
 		Attributes: map[string]schema.Attribute{
 			"id":   schema.StringAttribute{Computed: true},
 			"name": schema.StringAttribute{Required: true},
+			"description": schema.StringAttribute{
+				Computed: true,
+			},
+			"teams": schema.ListAttribute{
+				Computed:    true,
+				ElementType: types.StringType,
```

Comment:
Thank you for your contribution! We want to keep this provider as a mere reflection of the public API whenever possible. Can you please change teams to be a [list of API objects](https://developer.pagerduty.com/api-reference/51b21014a4f5a-list-escalation-policies) instead of a list of IDs? Also, for this to work you should add `Include: []string{"teams"}` to `pagerduty.ListEscalationPoliciesOptions` at line 53.

Reply:
> Also, for this to work you should add Include: []string{"teams"} to pagerduty.ListEscalationPoliciesOptions at line 53

I don't believe I should; `teams` is compute (and not exposed as a user-facing argument). It's intended to be the complete list of all teams returned by the API. Do you see things differently?

Reply:
> Also, for this to work you should add Include: []string{"teams"} to pagerduty.ListEscalationPoliciesOptions at line 53.

Done!

Reply:
Oh, great catch! Let's keep it this way then!